### PR TITLE
Add --xmloutput to vespa-get + vespa-visit

### PR DESCRIPTION
**Pull request by bjorncs**
@baldersheim OR @arnej27959 OR @vekterli

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]

File: vespaclient-java/src/main/java/com/yahoo/vespaget/CommandLineOptions.java [hunk 1]
```diff
@@ -216,7 +227,7 @@ public ClientParameters parseCommandLineArguments(String[] args) throws IllegalA
                     .setTraceLevel(trace)
                     .setPriority(priority)
                     .setTimeout(timeout)
-                    .setJsonOutput(jsonOutput)
+                    .setJsonOutput(!jsonOutput && !xmlOutput ? false : jsonOutput) // TODO Vespa 7 Change default to JSON
```
Comment (vekterli) [inline vespaclient-java/src/main/java/com/yahoo/vespaget/CommandLineOptions.java]: Some parenthesis-wrapping would be nice here to make operator precedence more unambiguous

File: vespaclient-java/src/main/java/com/yahoo/vespavisit/VdsVisit.java [hunk 2]
```diff
@@ -573,9 +580,13 @@ public VdsVisitParameters parse(String args[]) throws org.apache.commons.cli.Par
...
             }
-            if (line.hasOption("jsonoutput")) {
-                allParams.setJsonOutput(true);
+
+            boolean jsonOutput = line.hasOption("jsonoutput");
+            boolean xmlOutput = line.hasOption("xmloutput");
+            if (jsonOutput && xmlOutput) {
+                throw new IllegalArgumentException("Cannot combine both xml and json output");
             }
+            allParams.setJsonOutput(!jsonOutput && !xmlOutput ? false : jsonOutput); // TODO Vespa 7 Change default to JSON
```
Comment (vekterli) [inline vespaclient-java/src/main/java/com/yahoo/vespavisit/VdsVisit.java]: Same here, re: operator precedence

**Review by vekterli — COMMENTED** [review]
:+1:

**Comment by bjorncs** [comment]
@vekterli https://github.com/vespa-engine/vespa/pull/7312

**Pull request merged**